bin/hubot.cmd

```diff
@@ -0,0 +1,3 @@
+@echo off
+
+npm install && node_modules\.bin\hubot.cmd %*
```

Comment:
Out of interest, why does this run `npm install`? I don't know how/where Hubot is deployed/installed, but I'd assume the `npm install` is part of the deploy?